src/instruments/src/SD/Pages/Hyd/Hyd.tsx

```diff
@@ -0,0 +1,767 @@
+/*
+ * A32NX
+ * Copyright (C) 2020-2021 FlyByWire Simulations and its contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import './Hyd.scss';
+import ReactDOM from 'react-dom';
+import React, { useEffect, useState } from 'react';
+import { getRenderTarget, setIsEcamPage } from '../../../Common/defaults';
+import { SimVarProvider, useSimVar } from '../../../Common/simVars';
+
+setIsEcamPage('hyd_page');
+
+export const HydPage = () => {
+    // The FADEC SimVars include a test for the fire button.
+    const [Eng1N2] = useSimVar('TURB ENG N2:1', 'Percent', 1000);
+    const [Eng2N2] = useSimVar('TURB ENG N2:2', 'Percent', 1000);
+
+    const [greenPressure] = useSimVar('L:A32NX_HYD_GREEN_PRESSURE', 'psi', 500);
+    const [yellowPressure] = useSimVar('L:A32NX_HYD_YELLOW_PRESSURE', 'psi', 500);
+    const [bluePressure] = useSimVar('L:A32NX_HYD_BLUE_PRESSURE', 'psi', 500);
+
+    const [greenPumpPBStatus] = useSimVar('L:A32NX_OVHD_HYD_ENG_1_PUMP_PB_IS_AUTO', 'boolean', 500);
+    const [yellowPumpPBStatus] = useSimVar('L:A32NX_OVHD_HYD_ENG_2_PUMP_PB_IS_AUTO', 'boolean', 500);
+    const [bluePumpPBStatus] = useSimVar('L:A32NX_OVHD_HYD_EPUMPB_PB_IS_AUTO', 'boolean', 500);
+
+    const [yellowElectricPumpStatus] = useSimVar('L:A32NX_OVHD_HYD_EPUMPY_PB_IS_AUTO', 'boolean', 500);
```

Comment:
My feeling is that pump status should directly come from the XPUMP_ACTIVE simvars. For example if any logic starts a pump without changing button status, we won't see it active.  If yellow epump is activated by cargo door operation from ground crew, it won't appear on ecam currently. Is it expected?

Reply:
For the Yellow electrical pump you are right. However, the status of the push button is important too.

Reply:
Totally agree, but my point is that I already evaluate the button and compute the pump activation logic inside the WASM system. So I think there is no point doing it the same at different places UNLESS the ecam logic is different from the actual pump logic. 
Probably need to ask for this on discord :)
As an example, you should also check for blue pump override to have it showing on the ECAM when pressed, and this is already done and handled in the WASM module.  Or at least if it's done wrong e only have to update it in one place :)